### ISSUE TITLE
Allow passing start options

### ISSUE THREAD

**Issue by ekilah** [react-native-contrib/react-native-heyzap#7]
The Heyzap start options are an unsigned integer bitmask passed to `+ (void) startWithPublisherID:(NSString *)publisherID andOptions:(HZAdOptions)options`. I don't know what the code is doing with the options right now - it appears they are being returned as an array to the js that called the bridge's `start` method.

**Comment by hassankhan** [comment]
Yeah it was kind of just a test to see if constants were being exported back to JS, I'll be getting rid of that in the future.

**Comment by ekilah** [comment]
cool, yeah once those constants are exported and available to the JS environment it'll be easy to do!
